packages/repl/src/lib/Output/console/ConsoleLine.svelte

```diff
@@ -92,6 +92,24 @@
 		font-family: var(--sk-font-mono);
 	}
 
+	@media screen and (prefers-color-scheme: dark) {
```

Comment:
This wouldn't work well, because of the theme switcher having different theme than system. 

`Repl` component recieves a theme property(light or dark). I believe it can be passed through to ConsoleLine(or added to the context), applied to root element of this component with `class:dark={theme === 'dark'}`

Reply:
ok, i've updated it.